sematic/config/settings.py

```diff
@@ -410,6 +412,14 @@ def _apply_env_var_overrides(
             settings[var] = new_value
 
 
+def _apply_scopes_overrides(plugin_scopes: PluginScopes):
+    for scope in PluginScope:
+        if scope.value in os.environ:
+            logger.debug("Overriding scope %s from environment variables", scope.value)
+
+            plugin_scopes[scope] = os.environ[scope.value].split(",")
```

Comment:
This enables setting scopes from the CLI. This should have gone in the next PR but 🤷 

This is useful to deploy the server without a `settings.yaml` file. Users can do:

```
$ docker run ... -e STORAGE=sematic.plugins.storage.S3Storage ...
```

We can also set this env var by default in the deploy image.